index.html

```diff
@@ -0,0 +1,141 @@
+        </section>
+    </div>
+    <section class="main-program" id="main-program">
+        <div>
+            <h2 class="text-light">Main Program</h2>
+            <div class="orange-line"></div>
+        </div>
+        <ul>
+            <li class="list">
+                <i class="material-icons">school</i>
+                <h3>Lecture</h3>
+                <p class="text-light">Listen to speakers from various countries about the message of sharing and opening.
+                </p>
+            </li>
+            <li class="list">
+                <i class="material-icons">museum</i>
+                <h3>TEGS Exihibition</h3>
+                <p class="text-light">Appreciate various creations applying TEGS lisence of technologists, organized from tecnological center Owerri.
+                </p>
+            </li>
+            <li class="list">
+                <i class="material-icons">forum</i>
+                <h3>Forum</h3>
+                <p class="text-light">Try creating your own work using open source lisence rather than just watching.</p>
+            </li>
+            <li class="list">
+                <i class="material-icons">home_repair_service</i>
+                <h3>Workshop</h3>
+                <p class="text-light">Try creating your own work using open source lisence rather than just watching.</p>
+            </li>
+            <li class="list">
+                <i class="material-icons">attractions</i>
+                <h3>TEGS Ignite</h3>
+                <p class="text-light">Get the opportunities to network with TEGS affliates around the world, also after the summit.</p>
+            </li>
+        </ul>
+        <button>Join TEGS Global Summit 2022</button>
+        <article class="see-program text-light" id="see-program"><i class="material-icons">arrow_right_alt</i> click here to .<a href="#speakers" class="text-light"><b><u> SEE THE FEATURED SPEAKERS</u></b></a>
+        </article>
+    </section>
+    <!-- Featured Speakers -->
+    <section class="speakers hidden" id="speakers">
+    </section>
+    <section class="partners hidden" id="partners">
+        <div>
+            <h2 class="text-light">Partner</h2>
+            <div class="orange-line"></div>
+        </div>
+        <ul>
+            <li>Mozilla</li>
+            <li>Google</li>
+            <li>NAVER</li>
+            <li>Addidas</li>
+            <li>Mitchelin</li>
+        </ul>
+    </section>
+    <footer class="footer hidden" id="footer">
+        <div>
+            <i class="material-icons">school</i>
+            <h2>Technological Education Nigeria</h2>
+        </div>
+        <div>2022 Technological Education Nigeria.<br>Some rights reserved</div>
+    </footer>
+    <div class="see-more" id="see-more">
+        <div>See more</div>
+        <i class="material-icons">keyboard_arrow_down</i>
+    </div>
+    <div class="see-less hidden" id="see-less">
+        <div>See less</div>
+        <i class="material-icons">keyboard_arrow_up</i>
```

Comment:
- When I visit the home page on mobile view, the `'See More'` and `'See Less'` work the way they should.
- [ ] However, like the first reviewer mentioned when I am on the desktop view and I click the `'SEE THE FEATURED SPEAKER'` link, and I navigate to the mobile view of the home page, the `'See More'` button shows fewer contents on the page and the `'See Less'` button shows more contents on the page. Please fix this bug. 👍